test/unit/services/awsAdapter_spec.js

```diff
@@ -5,7 +5,9 @@ var expect     = require("chai").expect;
 var Bluebird   = require("bluebird");
 var Sinon      = require("sinon");
 var AWS        = require("aws-sdk");
-var ec2        = new AWS.EC2();
+var ec2        = new AWS.EC2({
+		region : "us-east-1"
```

Comment:
Can you explain the rationale for hard-coding a region here? I wouldn't expect this to be necessary (or desirable).